src/Ocelot/Multiplexer/MultiplexingMiddleware.cs

```diff
@@ -255,5 +255,27 @@ protected virtual Task MapAsync(HttpContext httpContext, Route route, List<HttpC
 
         var aggregator = _factory.Get(route);
         return aggregator.Aggregate(route, httpContext, contexts);
+    }
+
+    protected virtual async Task<Stream> CloneRequestBodyAsync(HttpRequest request, CancellationToken aborted)
+    {
+        request.EnableBuffering();
+        if (request.Body.Position == 0)
+        {
+            var targetBuffer = new MemoryStream();
+            if (request.ContentLength is not null)
+            {
+                await request.Body.CopyToAsync(targetBuffer, (int)request.ContentLength, aborted);
+                targetBuffer.Position = 0;
+                request.Body.Position = 0;
+            }

```

Comment:
```suggestion
            }
            else
            {
                Logger.LogWarning("Aggregation does not support body copy without Content-Length header!");
            }
```

Reply:
@raman-m it's done

I was on board for a test with a form file, but if it's enough for you I don't think it's necessary.